src/api/app/models/event/workflow_run_fail.rb

```diff
@@ -0,0 +1,17 @@
+module Event
+  class WorkflowRunFail < Base
+    self.message_bus_routing_key = 'workflow_run.fail'
+    self.description = 'Workflow run has failed'
+    payload_keys :id, :token_id
+
+    receiver_roles :token_executor
+
+    def token_executors
+      [Token.find_by(id: payload['token_id'], type: 'Token::Workflow')&.executor]
```

Comment:
```suggestion
      [Token.find_by(id: payload['token_id'], type: 'Token::Workflow')&.executor.compact]
```
In order to remove the nil that will appear if there is no executor or no token

Reply:
Applied in two places